hermes-client/src/main/java/pl/allegro/tech/hermes/client/webclient/WebClientHermesSender.java

```diff
@@ -24,10 +25,10 @@ public WebClientHermesSender(WebClient webClient) {
     }
 
     @Override
-    public CompletableFuture<HermesResponse> send(URI uri, HermesMessage message) {
+    public Mono<HermesResponse> sendReactively(URI uri, HermesMessage message) {
         return webClient.post()
                 .uri(uri)
-                .syncBody(message.getBody())
+                .syncBody(message.getBody()) // TODO reactively?
```

Comment:
hmm this is probably fine, because you already have the body of the message, if you'd have the body as the Mono, it'd be the case ;)